runtime/renderer/xwalk_content_renderer_client.cc

```diff
@@ -31,6 +32,7 @@
 
 #if defined(OS_ANDROID)
 #include "components/cdm/renderer/android_key_systems.h"
+#include "xwalk/runtime/common/android/xwalk_render_view_messages.h"
```

Comment:
Better put this below #include "xwalk/runtime/browser/android/net/url_constants.h"